### ISSUE TITLE
still completing obby for luck boost when the boost didnt respawn

### ISSUE THREAD

**Issue by smge0** [BuilderDolphin/dolphSol-Macro#211]
shortly check for obby buff effect doesnt work.

**Comment by kuromian** [comment]
I agree with this, but I'm assuming you just started your macro again so that's most likely why that happened.

**Comment by smge0** [comment]
> I agree with this, but I'm assuming you just started your macro again so that's most likely why that happened.

even if i once start it it repeats the same issue
